go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -442,6 +458,22 @@ func (c *TabletConfig) verifyTransactionLimitConfig() error {
 	return nil
 }
 
+// verifyTxThrottlerConfig checks the TxThrottler related config for sanity.
+func (c *TabletConfig) verifyTxThrottlerConfig() error {
+	if len(c.TxThrottlerTabletTypes) == 0 {
+		return vterrors.New(vtrpcpb.Code_FAILED_PRECONDITION, "--tx-throttler-tablet-types must be defined when transaction throttler is enabled")
+	}
+	for _, tabletType := range c.TxThrottlerTabletTypes {
+		switch tabletType {
+		case topodatapb.TabletType_REPLICA, topodatapb.TabletType_RDONLY:
+			continue
+		default:
+			return vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "unsupported tablet type %q", tabletType)
```

Comment:
Consider making the error message be more explicit about the problem being with the tablet type for the tx throttler.

Reply:
@ejortegau good point 👍. This matches upstream so I'd prefer to keep this PR as-is, but I'll fix this in an upcoming (hopefully final) cleanup PR

Would `txThrottler: tablet type %q is not one of supported types replica and rdonly` work?

Reply:
Sounds good